docs/getting-started/integrations/angular.md

```diff
@@ -92,15 +92,7 @@ All available Angular versions are [listed on npm](https://www.npmjs.com/package
 
 ## Quick start
 
-### Using CKEditor&nbsp;5 Builder
-
-The easiest way to use CKEditor 5 in your Angular application is by configuring it with [CKEditor&nbsp;5 Builder](https://ckeditor.com/builder?redirect=docs) and integrating it with your application. Builder offers an easy-to-use user interface to help you configure, preview, and download the editor suited to your needs. You can easily select:
-
-* the features you need,
-* the preferred framework (React, Angular, Vue or Vanilla JS),
-* the preferred distribution method.
-
-You get ready-to-use code tailored to your needs!
+{@snippet getting-started/use-builder}
```

Comment:
I like this new section. I don't know if I can articulate my problem, but something is off with this placement. It may be because similar headings one under the other look strange. Maybe we should move it above the "Quick Start"? That would even work with the flow because the builder is not required to go through the tutorial. Here's how it would look:
<img width="1010" alt="Screenshot 2024-09-05 at 09 40 55" src="https://github.com/user-attachments/assets/4fb9f4f2-f5cf-4776-814e-134345234460">
@godai78 @filipsobol WDYT?

Reply:
It may also be because of stacked borders.

Reply:
Moving it up looks logical, but won't it disturb the flow? Like, you enter the lift and there's stairs inside.

Reply:
Changed :heavy_check_mark: